flink-table/flink-table-planner/src/main/java/org/apache/flink/table/planner/operations/SqlToOperationConverter.java

```diff
@@ -972,20 +987,21 @@ private Operation convertShowViews(SqlShowViews sqlShowViews) {
 
     /** Convert RICH EXPLAIN statement. */
     private Operation convertRichExplain(SqlRichExplain sqlExplain) {
-        Operation operation;
+        List<Operation> operations = new ArrayList<>();
```

Comment:
No need to instantiate this list, just assign operations in the if branches with either a singleton list or the list returned by `convertSqlStatementSet`.